app/jobs/regular/group_smtp_email.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require_dependency 'email/sender'
```

Comment:
Just being curious, why was it removed? Wasn't needed?

Reply:
Indeed it’s not needed anymore as we’re autoloading what we have in `lib`. We already did that with Rails 6+ but I think our configuration wasn’t working as expected and that’s why we needed all those `require_dependency` calls.